### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/canonical_json.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Issue with canonical_json
+about: Bug, feature request or question regarding 'package:canonical_json'
+title: ''
+labels: pkg:canonical_json, pending-triage
+assignees:
+---
+ * Explain the issue,
```
Comment (sigurdm) [inline .github/ISSUE_TEMPLATE/canonical_json.md]: Would you not want to guide the user a bit more?

What versions of everything where you using?
What did you expect?
What did you see?

File: tool/update-github-config.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```
Comment (sigurdm) [inline tool/update-github-config.sh]: Why not dart?? :)

File: tool/update-github-config.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+LABELS=`hub api -X GET '/repos/google/dart-neats/labels' | jq -r '.[].name' | grep 'pkg:'`
+
+for P in $PACKAGES; do
+  if `list_include_item "$LABELS" "pkg:$P"`; then 
+    echo " - skip pkg:$P"
+  else 
+    echo " - creating: pkg:$P"
+    hub api -X POST '/repos/google/dart-neats/labels' -F name="pkg:$P" -F color=f29513 | jq
+  fi
+done
```
Comment (sigurdm) [inline tool/update-github-config.sh]: Missing final newline

File: tool/update-github-config.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+\`\`\`
+EOF
+done
+
+function list_include_item {
+  local list="$1"
+  local item="$2"
+  if [[ $list =~ (^|[[:space:]])"$item"($|[[:space:]]) ]] ; then
+    # yes, list include item
+    result=0
```
Comment (sigurdm) [inline tool/update-github-config.sh]: `return 0`?

Comment (jonasfj) [reply tool/update-github-config.sh]: Yeah, this could be cleaner :)

File: tool/update-github-config.sh (hunk 2 above)
Comment (jonasfj) [reply tool/update-github-config.sh]: Shelling out to `hub` and `git ls-files` is easier to bash... it's just a quick and dirty script, I wasn't even sure if it was worth checking in :)

File: .github/ISSUE_TEMPLATE/canonical_json.md (hunk 1 above)
Comment (jonasfj) [reply .github/ISSUE_TEMPLATE/canonical_json.md]: Not for now, people tend to follow templates too much.. I want to see where this is going first :)

Mostly, I want issues to be labelled by package, and then I don't want more templates than that.

**Pull request merged**